src/python/nxt_python_asgi_lifespan.c

```diff
@@ -11,6 +11,7 @@
 #include <nxt_main.h>
 #include <python/nxt_python_asgi.h>
 #include <python/nxt_python_asgi_str.h>
+#include <structmember.h>
```

Comment:
Hmm, what is this? Oh, it's actually from Python...

Maybe a blank line before it to show it's separate from the internal Unit includes.